core/src/mindustry/entities/abilities/MovementLightningAbility.java

```diff
@@ -0,0 +1,47 @@
+package mindustry.entities.abilities;
+
+import arc.graphics.*;
+import arc.math.*;
+import arc.util.*;
+import arc.audio.*;
+import mindustry.content.*;
+import mindustry.entities.*;
+import mindustry.gen.*;
+
+public class MovementLightningAbility extends Ability{
+    //Lightning damage
+    public float damage = 35f;
+    //Ticks between firing
+    public float reload = 15f;
+    //Length of lightning
+    public int length = 12;
+    //Speeds for when to start lightninging and when to stop getting faster
+    public float minSpeed = 0.8f, maxSpeed = 1.2f;
+    //Shoot effect
+    public Effect shootEffect = Fx.sparkShoot;
```

Comment:
Yes, I can see that the variable is called "shootEffect", you don't need to write "Shoot effect" above it. Either add more descriptive comments (*"effect created at unit position when shooting"*), or don't add anything. Same for the sound.